testgen/src/tester.rs

```diff
@@ -305,9 +431,17 @@ impl Tester {
                 Ok(paths) => {
                     for path in paths {
                         if let Ok(entry) = path {
+                            // ignore path components starting with '_'
+                            if let Some(last) = entry.path().iter().rev().next() {
+                                if let Some(last) = last.to_str() {
+                                    if last.starts_with('_') {
+                                        continue;
+                                    }
+                                }
+                            }
```

Comment:
This code could be extracted into a helper function to improve readability.

Reply:
done